crates/proto/src/dnssec/rdata/tsig.rs

```diff
@@ -800,16 +797,14 @@ pub fn signed_bitmessage_to_buf(
 }
 
 /// Helper function to make a TSIG record from the name of the key, and the TSIG RData
-pub fn make_tsig_record(name: Name, rdata: TSIG) -> Record {
+pub fn make_tsig_record(name: Name, rdata: TSIG) -> Record<TSIG> {
     // https://tools.ietf.org/html/rfc8945#section-4.2
 
     let mut tsig = Record::from_rdata(
         //   NAME:  The name of the key used, in domain name syntax
-        name,
-        //   TTL:  This MUST be 0.
-        0,
-        //   TYPE:  This MUST be TSIG (250: Transaction SIGnature).
-        DNSSECRData::TSIG(rdata).into(),
+        name, //   TTL:  This MUST be 0.
```

Comment:
I think these comments ended up shifting awkwardly where the TTL comment is next to the `name` binding, and the TYPE comment is next to the TTL value.